### PR TITLE
First pass at accessibility improvements

### PR DESCRIPTION
**Pull request by sedders123**
These changes are primarily led by following suggestions from Google's [Lighthouse](https://developers.google.com/web/tools/lighthouse) tool. While we still don't have a perfect score (and the Lighthouse tool itself isn't perfect) these changes are meant to be non controversial and are definite steps in the right direction. Larger changes or changes that may lead to more comments will be done in separate PRs.

### CODE REVIEW

File: _events/sr2020/virtual-competition-knockouts.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ Code submission will be due at 10am BST: [Submit your code](https://studentrobot
 
 Our live-stream will start at 3pm BST: [Watch live](https://youtu.be/xBPVqsb_Ydk)
 
-<iframe width="100%" height="315" src="https://www.youtube.com/embed/xBPVqsb_Ydk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe title="Livestream of the Knockouts and Final" width="100%" height="315" src="https://www.youtube.com/embed/xBPVqsb_Ydk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (raccube) [inline _events/sr2020/virtual-competition-knockouts.md:15]: Livestream is written both with and without a dash on this page.

**Comment by raccube** [comment]
Thought: Could we somehow catch accessibility issues in CI?

**Comment by PeterJCLaw** [comment]
> Thought: Could we somehow catch accessibility issues in CI?

Yes: https://github.com/marketplace/actions/lighthouse-ci-action exists and has a recipe for this (look for "Test a static site without having to deploy it.")

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@ <h2 class="text-center">What is Student Robotics?</h2>
         <img srcset="{{ '/images/advertising/team-in-arena@1x.jpg 1x' | prepend: site.baseurl }},
                       {{ '/images/advertising/team-in-arena@2x.jpg 2x' | prepend: site.baseurl }},
                       {{ '/images/advertising/team-in-arena@3x.jpg 3x' | prepend: site.baseurl }}"
-              src="{{ '/images/advertising/team-in-arena@3x.jpg' | prepend: site.baseurl }}"
+          src="{{ '/images/advertising/team-in-arena@3x.jpg' | prepend: site.baseurl }}"
```
Comment (PeterJCLaw) [inline index.html:12]: Whitespace change seems unrelated? Also seems to have changed the indentation away from lining up with the other attribute.

I suggest if we're going to go for a fixed-indent style rather than the current align-indent style that we wrap all the attributes:
``` html
<foo
  attr="blah"
  attr2="blah"
/>
```

File: _sass/testimonials.scss [hunk 3]
```diff
@@ -13,3 +13,9 @@
     }
   }
 }
+
+.testimonial-footer {
+  color: $sr-blue;
+  font-family: "Open Sans Condensed", sans-serif;
+  text-transform: uppercase;
```
Comment (PeterJCLaw) [inline _sass/testimonials.scss:20]: This feels really easy to miss if we changed our heading styles. Maybe extract a heading style mixin and use that here?

File: index.html [hunk 4]
```diff
@@ -209,7 +209,8 @@ <h2>Our sponsors</h2>
 <div class="panel">
   <div class="fixed-width text-center">
     <h2>Donate</h2>
-    <p>Every donation is greatly appreciated, no matter the amount, and directly contributes to us running the competition for our teams.</p>
+    <p>Every donation is greatly appreciated, no matter the amount, and directly contributes to us running the
+      competition for our teams.</p>
```
Comment (PeterJCLaw) [inline index.html:213]: Unrelated layout change?

File: _events/sr2019/competition.md [hunk 5]
```diff
@@ -53,14 +53,14 @@ Media consent forms will also be available at reception.
...
-United Kingdom  
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2514.4224296392836!2d-1.399433684090786!3d50.934400360256014!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487473f7089148ab%3A0xaeeea75e9ec72c88!2sUniversity+of+Southampton+Students&#39;+Union+(SUSU)!5e0!3m2!1sen!2suk!4v1551272133022" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+Student's Union
+University of Southampton
+University Road
+Southampton
+SO17 1BJ
+United Kingdom
+
```
Comment (PeterJCLaw) [inline _events/sr2019/competition.md:62]: This whitespace change is going to be breaking -- the trailing whitespace here is important to mark that these lines should be wrapped as they are.

**Review by PeterJCLaw — CHANGES_REQUESTED** [review]
Nice one :+1:
This looks good, though unfortunately I think the change in `_events/sr2019/competition.md` will break its layout of the address.

File: _events/sr2020/virtual-competition-knockouts.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sedders123) [reply _events/sr2020/virtual-competition-knockouts.md:15]: 👍 Standardised in #380

**Review by PeterJCLaw — APPROVED** [review]

**Review by raccube — APPROVED** [review]
